vllm/config.py

```diff
@@ -3043,13 +3050,36 @@ def compute_hash(self) -> str:
     "bfloat16": torch.bfloat16,
 }
 
-_ROCM_NOT_SUPPORTED_DTYPE: list[str] = []  #
+# model_type -> reason
+_FLOAT16_NOT_SUPPORTED_MODELS = {
+    "gemma3": "Numerical instability. Please use bfloat16 or float32 instead.",
```

Comment:
```suggestion
    "gemma2": "Numerical instability. Please use bfloat16 or float32 instead.",
    "gemma3": "Numerical instability. Please use bfloat16 or float32 instead.",
```

I remember `gemma2` also has numerical instability issue like gemma3.